.github/workflows/go.yml

```
@@ -13,25 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.x
-      uses: actions/setup-go@v2
-      with:
-        go-version: ^1.16
-
-    - name: Setup Node.js environment
-      uses: actions/setup-node@v2.1.2
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-    
-    - name: Setup Task
-      run: curl -sL https://taskfile.dev/install.sh | sh
-    
-    - name: Build using Taskfile.yml
-      run: ./bin/task build
-    
-    - name: Archive production artifacts
-      uses: actions/upload-artifact@v2
-      with:
-        name: compiled-application-amd64-ubuntu
-        path: out/*
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.16
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
```

I think the name here is not correct :thinking: why do you say the code is checked out into the Go module directory? I think is checked out as any other job using `actions/checkout`.
```suggestion
      - uses: actions/checkout@v2
```

---

Good question. I guess at the time of writing this name I just wanted to add one. Never the less this name is gonna be removed too. 🤷